code/modules/mob/living/living.dm

```diff
@@ -286,6 +286,10 @@
 		visible_message("<span class='danger'>[src] points [trick] at- and [trick] goes off in their hand!</span>")
 		trick.shoot_gun(src)
 
+	if(istype(hand_item, /obj/item/slapper))
```

Comment:
Assuming you only want it to happen while the slapper is in your active hand while you point to it, and not just any time you're holding a slapper as your active hand and you point
```suggestion
	if(istype(hand_item, /obj/item/slapper) && hand_item == A)
```